### ISSUE TITLE
Feature Review - Detail/Player Toggle

### ISSUE THREAD

**Issue by jordanwillis** [phantomshadowghostshadowghost/Jammming_2#1]
Great job clearly documenting a really nice new feature!  I think the ability for users to hear the song from the app would definitely be a nice addition to the Jammming app.

In general, you did a nice job explaining each aspect of the feature starting from a brief description in the Objective section all the way to the full technical design.  One thing that I would recommend is ensure you focus on the overall goal of the feature in the Objective section, instead of immediately discussing the proposed implementation.  Here is an example of what I mean...

I would say that the true Objective of this feature is "To give users the ability to listen to each track, in addition to viewing song and artist detail, so they can decide if they want to save the track to a playlist"

In other words, speak from the user's perspective of what they want to do and use the rest of the document to highlight how you can make that happen.  This is critically important because it keeps the user at the center of the discussion and enables project decision makers to assess if the approach laid out in the rest of the document is the best way to meet the goal.  For example, perhaps they might decide that instead of adding a player toggle button, they want to just directly add a play button instead.
